### PR TITLE
Updating docker docs to include all inputs

### PR DESCRIPTION
**Pull request by ricky-undeadcoders**

### CODE REVIEW

**Review by zeitlinger — APPROVED** [review]

**Review by zerok — APPROVED** [review]

File: actions/build-push-to-dockerhub/README.md [hunk 1]
```diff
@@ -31,13 +31,14 @@ jobs:
...
-| `repository`| String | Docker repository name |
-| `tags` | List | Tags that should be used for the image (see the [metadata-action][mda] for details) |
+| Name         | Type   | Description                                                                         |
+|--------------|--------|-------------------------------------------------------------------------------------|
+| `context`    | String | Path to the Dockerfile (default: `.`)                                               |
+| `platforms`  | List   | List of platforms the image should be built for (e.g. `linux/amd64,linux/arm64`)    |
+| `push`       | Bool   | Push the generated image (default: `false`)                                         |
+| `repository` | String | Docker repository name                                                              |
+| `tags`       | List   | Tags that should be used for the image (see the [metadata-action][mda] for details) |
+| `file`       | String | Path and filename of the dockerfile to build from.                                  |
```
Comment (jdbaldry) [inline actions/build-push-to-dockerhub/README.md:41]: Given https://github.com/grafana/shared-workflows/pull/47#issuecomment-1960907637, I think it's useful to tell people that this path is resolved relative to the workspace and not the context directory.

Comment (zerok) [inline actions/build-push-to-dockerhub/README.md:41]:
```suggestion
| `file`       | String | Path and filename of the dockerfile to build from. (Default: `{context}/Dockerfile`) |
```

File: actions/push-to-gar-docker/README.md [hunk 2]
```diff
@@ -33,3 +33,15 @@ jobs:
...
+
+| Name          | Type   | Description                                                                         |
+|---------------|--------|-------------------------------------------------------------------------------------|
+| `registry`    | String | Google Artifact Registry to store docker images in.                                 |
+| `tags`        | List   | Tags that should be used for the image (see the [metadata-action][mda] for details) |
+| `context`     | List   | Path to the Docker build context.                                                   |
+| `environment` | Bool   | Environment for pushing artifacts (can be either dev or prod).                      |
+| `image_name`  | String | Name of the image to be pushed to GAR.                                              |
+| `build-args`  | String | List of arguments necessary for the Docker image to be built.                       |
+| `file`        | String | Path and filename of the dockerfile to build from.                                  |
```
Comment (zerok) [inline actions/push-to-gar-docker/README.md:47]:
```suggestion
| `file`        | String | Path and filename of the dockerfile to build from. (Default: `{context}/Dockerfile`) |
```

**Review by zerok — CHANGES_REQUESTED** [review]
LGTM :) Just these two little changes that came in through https://github.com/grafana/shared-workflows/pull/50

**Pull request closed without merging**